app/services/config_report_importer.rb

```diff
@@ -0,0 +1,41 @@
+class ConfigReportImporter < ReportImporter
+  attr_reader :report
+  def self.authorized_smart_proxy_features
+    ['Puppet']
+  end
+
+  def report_name_class
+    ConfigReport
+  end
+
+  def import
+    @report = super
```

Comment:
It's set here though, isn't it?  https://github.com/theforeman/foreman/pull/2530/files#diff-5f2522f8937b1076938bcca2ab931cf1R50